cli/test/test-frame-fit-android-shared.mjs

```
@@ -0,0 +1,289 @@
+    fn()
+    passed++
+    console.log(`✔ ${name}`)
+  }
+  catch (error) {
+    failed++
+    console.error(`✖ ${name}\n  ${error.message}`)
+  }
+}
+
+const h = React.createElement
+const noop = () => {}
```

ESLint

[error] Top-level functions should be declared with function keyword

(antfu/top-level-function)